ttmar_starter.html

```diff
@@ -61,5 +61,66 @@
 
 </body>
 <script>
+var firstEvent = document.querySelector("#first_paragraph");
+var firstBlock = document.querySelector(".first_paragraph");
 
+firstEvent.addEventListener("click", function(){
+  console.log("hi");
+  firstBlock.style.background= "#000";
+})
+
+
+var secondEvent = document.querySelector("#second_paragraph");
+var secondBlock = document.querySelector(".second_paragraph");
+
+secondEvent.addEventListener("click", function(){
+  secondBlock.style.display="block";
+})
+
+
+var thirdEvent = document.querySelector("#first_chorus");
+var thirdBlock = document.querySelector(".first_chorus");
+
+thirdEvent.addEventListener("click", function(){
+  event.preventDefault();
+  thirdBlock.style.display="block";
+})
+
+
+var fourthEvent = document.querySelector("#third_paragraph");
+var fourthBlock = document.querySelector(".third_paragraph");
+
+fourthEvent.addEventListener("click", function(){
+  event.preventDefault();
+  for (i=0; i < 4; i++){
```

Comment:
From here (fourthEvent, etc) to the end was all from the solution code.  I kept looking for a way to select all the ".third_paragraph" elements all at once because I didn't know they behaved like an array; and because I didn't know they were an array, I didn't think of iterating through them with a loop.  I was completely stumped.